test/cart.test.js

```diff
@@ -39,51 +40,19 @@ describe('cart / saveCart', () => {
 
     await saveCart(cart, {}, {})
 
-    sinon.assert.calledOnceWithMatch(global.window._recart.setCart,
-      {
-        itemCount: 0,
-        value: 0,
-        returnUrl: global.window.location.hostname
-      },
-      [],
-      fakedNow
-    )
+    expect(global.window._recart.setCart.callCount).to.equal(1)
```

Comment:
calledOnceWithExactly? It would be better.

Reply:
I prefer expect over assert as it gives a more readable comparison upon failure.